zuul.d/jobs.yaml

```diff
@@ -7,3 +7,11 @@
 - job:
     name: eib-crc-podified-edpm-baremetal
     parent: cifmw-crc-podified-edpm-baremetal
+    dependencies: ["eib-content-provider-build-images"]
+
+- job:
+    name: eib-podified-multinode-ironic-deployment
+    parent: podified-multinode-ironic-deployment
+    dependencies: ["eib-content-provider-build-images"]
+    vars:
+      cifmw_update_containers_ipa_image_url: "{{ cifmw_build_images_output['images']['ironic-python-agent']['image'] }}"
```

Comment:
Does podified-multinode-ironic-deployment deploy any edpm nodes? If so we should use the built edpm image too

Reply:
@steveb Thank you for reviewing the pr. eib-crc-podified-edpm-baremetal is deploying edpm node using built uefi image. podified-multinode-ironic-deployment  will use the built ipa image. these two jobs will test both edpm images. I hope it answers the above question.